photon-ml/src/main/scala/com/linkedin/photon/ml/model/GAMEModel.scala

```diff
@@ -22,29 +22,37 @@ import org.apache.spark.storage.StorageLevel
 import scala.collection.Map
 
 /**
- * Representation of the generalized additive mixed effect (GAME) model
+ * Generalized additive mixed effect (GAME) model
  *
- * @param gameModels A (modelName -> model) map representation of the models that consist of the GAME model
+ * @param gameModels A (modelName -> model) map containing the models that make up the complete GAME model
  */
 class GAMEModel(gameModels: Map[String, DatumScoringModel]) extends DatumScoringModel {
 
+  require(gameModels != null)
```

Comment:
Yep, I'm with you on spelling out preconditions, but use of "null" is highly discouraged in Scala because it's supposed to be the compiler's job to guarantee that references are right. If we have cases where a null is passed here, that's definitely a bug in the calling code and should be removed.